src/components/AccountManagementLayout/PasswordStrength.tsx

```diff
@@ -7,20 +7,32 @@ interface PasswordStrengthProps {
 }
 export default class PasswordStrength extends Component<PasswordStrengthProps> {
   createPasswordLabel(result: ZXCVBNResult) {
+    let x;
```

Comment:
I don't think it is necessary to say what the crack time is. After looking at my other comment, I would revert these changes as saying "Strong" should be plenty since the description says you need a "Strong" password.

Reply:
So just have the old meter and have the password guidelines or do you want to have it show the suggestions still just not the warnings?